_layouts/doc.html

```diff
@@ -86,6 +85,9 @@
 							<li><a href="{{ site.baseurl }}/docs/ui-toolkit/components/headers">Headers</a></li>
 							<li><a href="{{ site.baseurl }}/docs/ui-toolkit/components/overlay">Overlay</a></li>
 							<li><a href="{{ site.baseurl }}/docs/ui-toolkit/components/text-input">TextInput</a></li>
+							<li><a href="{{ site.baseurl }}/docs/ui-toolkit/components/video">Video</a></li>
+							<li><a href="{{ site.baseurl }}/docs/ui-toolkit/components/lightbox">Lightbox</a></li>
+							<li><a href="{{ site.baseurl }}/docs/ui-toolkit/components/rich-media">Rich Media</a></li>
```

Comment:
Done. Updated `NavigationBar`, `DropDownMenu`, `ListView` and `GridView` to be without space, to be consistent with official RN docs.